portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 5) + 1;
+  const imgUrl = 'images/jason-' + imageIndex + '.jpg';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
+
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
```

Comment:
I'm not sure what modalbtnX are used for. But might be helpful to have more descriptive names, like what the modal is like InterestsModal, IntroModal, etc. Same for your element ID's.